hudi-common/src/main/java/org/apache/hudi/common/table/log/AbstractHoodieLogRecordReader.java

```diff
@@ -167,7 +167,7 @@ protected AbstractHoodieLogRecordReader(HoodieStorage storage, String basePath,
     this.payloadClassFQN = tableConfig.getPayloadClass();
     this.preCombineField = tableConfig.getPreCombineField();
     // Log scanner merge log with precombine
-    TypedProperties props = new TypedProperties();
+    TypedProperties props = hoodieTableMetaClient.getTableConfig().getProps();
```

Comment:
Revert changes in `AbstractHoodieLogRecordReader.java` as we do not want to introduce potential breaking changes in the existing log record reader.

Reply:
It's better to strictly keep the required props here, i.e., the ordering field prop.

Reply:
done.